app/modules/ia_config_reader.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import json
+import os.path as path
+
+log = logging.getLogger(__name__)
+
+
+def load_config_to_dict(fname):
+    app_home = ''
+    config_path = path.join(app_home, 'ia-configs', fname)
+    assert path.isfile(config_path), f'Could not find config at path {config_path}'
+    with open(config_path, 'r') as file:
```

Comment:
How will this handle the case where the file is not present? It should handle it gracefully.

Reply:
If we try to construct an ia config reader for a file that doesn't exist it should fail, right? What other behavior would be more graceful?

Reply:
Fair point, I still occasionally look at assert and think C style assert which is extremely ungraceful. Python assert is the right choice here